go.mod

```diff
@@ -34,7 +34,6 @@ require (
 	cosmossdk.io/api v0.2.6 // indirect
 	cosmossdk.io/core v0.5.1 // indirect
 	cosmossdk.io/depinject v1.0.0-alpha.3 // indirect
-	cosmossdk.io/errors v1.0.0-beta.7 // indirect
```

Comment:
cosmossdk.io/errors is a direct import after https://github.com/cosmos/gaia/pull/2471

Reply:
This will go away once https://github.com/cosmos/gaia/pull/2592 is merged